warcio/cli.py

```diff
@@ -93,6 +105,22 @@ def decompress_and_recompress(self, stream, output):
             tout.seek(0)
             self.load_and_write(tout, output)
 
+# ============================================================================
+def extract_record(cmd):
+    records = cmd.records.split(',')
+    writer = WARCWriter(filebuf=sys.stdout.buffer, gzip=False)
+    for filename in cmd.inputs:
+        with open(filename, 'rb') as fh:
+            for record in ArchiveIterator(fh, no_record_parse=True, arc2warc=True):
```

Comment:
Do you want to have `arc2warc` be true? If it is true, the ARC record is converted on the fly to an WARC record, so only the 2nd condition would be needed. But, you may want to print the original ARC record here when extracting and not convert.